Dockerfile

```diff
@@ -17,7 +17,7 @@ WORKDIR /opt/
 COPY requirements.txt requirements.txt
 
 # Download the variant_definitions in the builder so that runner doesn't need wget
-RUN wget -q "https://raw.githubusercontent.com/NebraLtd/helium-hardware-definitions/master/variant_definitions.py"
+RUN wget -q "https://raw.githubusercontent.com/NebraLtd/helium-hardware-definitions/d84ee2af049b9c033c07788bea577a459aeb3aaa/variant_definitions.py"
```

Comment:
@vpetersson I actually don't like this either. I think this file should be copied into source control.

Reply:
@marvinmarnold IIRC the reason it is in a separate repo is because it is also used in https://github.com/NebraLtd/hm-pktfwd/blob/master/Dockerfile and possibly elsewhere. However, not sure if that makes sense or not

Reply:
What we should really be doing is publishing that file to a python library that can be installed with `pip`. Until we are at that level, I think we should manually copy into a lib/ directory so that the local development environment has access. Something like this: https://github.com/NebraLtd/hm-config/pull/54/commits/b5e08e48e05d58adb9681e9b3e182d1cdcd771d8